pydantic_evals/pydantic_evals/evaluators/llm_as_a_judge.py

```diff
@@ -221,39 +221,44 @@ def _stringify(value: Any) -> str:
         return repr(value)
 
 
+def _add_section(sections: list[str | UserContent], content: Any, tag: str) -> None:
+    """Add a tagged section *inplace* to the sections list, handling different content types."""
```

Comment:
There doesn't seem to be a particular reason why this needs to add to an existing list, so I'd prefer to just have it return a list that we can then combine at the call site